geowebcache/pom.xml

```diff
@@ -53,32 +53,43 @@
   
   <repositories>
     <repository>
-      <id>boundless</id>
-      <name>Boundless Maven Repository</name>
-      <url>https://repo.boundlessgeo.com/main/</url>
+      <id>osgeo-release</id>
+      <name>OSGeo Repository</name>
```

Comment:
Maybe use "OSGeo Release Repository: for the name here (and "OSGeo Snapshot Repository" below)